mock_service/lib/dsl.py

```diff
@@ -143,3 +159,18 @@ def create_animation_action(
         AnimationAction: The created AnimationAction.
     """
     return AnimationAction(duration, repeat_mode, values, __resolve_value)
+
+def create_EventTrigger(type: EventType, path: Optional[str] = None) -> EventTrigger:
```

Comment:
Does not follow established naming conventions: should be called `create_event_trigger`